scout/load/hgnc_gene.py

```diff
@@ -78,20 +95,36 @@ def load_hgnc_genes(
             genemap_lines=genemap_lines,
         )
 
-    non_existing = 0
+    without_coords = 0
     nr_genes = len(genes)
     LOG.info(f"Building info for {nr_genes} genes")
+
+    cytoband_coords: Dict[str, dict] = adapter.cytoband_to_coordinates(build=build)
+
     with progressbar(genes.values(), label="Building genes", length=nr_genes) as bar:
         for gene_data in bar:
+            set_gene_coordinates(gene_data=gene_data, cytoband_coords=cytoband_coords)
+
             if not gene_data.get("chromosome"):
```

Comment:
Note that there is also this other check to make sure that coords will not be missing